doc/source/train/distributed-pytorch/quickstart.rst

```diff
@@ -0,0 +1,14 @@
+Quickstart
```

Comment:
There's a decent amount of duplication in
- Ray Train Landing Page
- Ray Train Getting Started
- Torch Quickstart
- Torch Converting an existing training loop

I do think we want to eventually de-dupe/clean these up, but in the short term should we hide this page? I think in its current state its more detracting than helpful

Reply:
Removed the getting started page. But yes, I think we can get rid of torch quickstart as well. The we only have the Ray Train start page for a very minimal example (and all libraries) and otherwise the full torch guide.